src/api/app/models/package.rb

```diff
@@ -1034,6 +1035,14 @@ def remove_linked_packages
     BackendPackage.where(links_to_id: self.id).delete_all
   end
 
+  def remove_devel_packages
+    Package.where(develpackage: self).each do |devel_package|
+      devel_package.develpackage = nil
```

Comment:
On Montag, 19. Oktober 2015, 01:38:19 CEST wrote Henne Vogelsang:
> > @@ -1034,6 +1035,14 @@ def remove_linked_packages
> >      BackendPackage.where(links_to_id: self.id).delete_all
> >    end
> >  
> > +  def remove_devel_packages
> > +    Package.where(develpackage: self).each do |devel_package|
> > +      devel_package.develpackage = nil
> 
> Broken Database? The links in the package that points to self should be broken and not silently discarded IMHO

After local discussion: This is restoring the old behaviour which was okay so far
for Factory (where this is mainly used). 

We could handle this different, for example like inventing a "deleted" package similar
to the "deleted" repository handling. But that would be another patch